src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs

```diff
@@ -461,5 +461,16 @@ public void ClickCheckbox(int index = 0)
 			Wait.Until(d => NumberOfCheckboxesDisplayed() > index);
 			Driver.FindElements(Pages.Common.Checkbox)[index].Click();
 		}
+
+		public bool IsCheckboxChecked(int index = 0)
+		{
+			Wait.Until(d => NumberOfCheckboxesDisplayed() > index);
+			return Driver.FindElements(Pages.Common.Checkbox)[index].GetAttribute("checked") != null;
```

Comment:
is this an attribute? `checked` can sometimes be referenced via `.GetProperty("checked")` if this doesn't work

Reply:
it appears to work locally. What is the difference between an attribute and a property?

Reply:
Attributes display directly on the DOM, properties are actionable parts of elements. The `checked` attribute/property is a little weird as it can transition between both...

Reply:
https://stackoverflow.com/questions/6003819/what-is-the-difference-between-properties-and-attributes-in-html#:~:text=Property%20values%20can%20change%3B%20attribute,property%20initialized%20to%20%22Bob%22.&text=The%20HTML%20attribute%20value%20specifies,property%20is%20the%20current%20value.

Reply:
So I tried property, and it appears to be returning true even after deselecting. so I'm going to stick with attribute in this case. Good to know going forward tho